src/argilla/client/feedback/dataset.py

```diff
@@ -11,10 +11,11 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-
+import json
```

Comment:
I guess we could maybe use `orjson` instead as it's faster? https://github.com/ijl/orjson

Reply:
Unless this is just temporary, in that case I guess we're good